pyproject.toml

```diff
@@ -336,6 +338,14 @@ line-length = 100
 target-version = "py310"
 
 [tool.ruff.lint]
+external = [
+  "DOC" # pydoclint
+]
+ignore = [
+  # incompatible with ruff formatter
```

Comment:
Untill we are using the ruff formatter are these necessary?

Reply:
I just enabled it, but apparently it does not do as much as black and I really wanted to force black to reformat or f-strings, they become quite popular and I was surprised too see that it did not touch a too long f-string today.